internal/chain/beacon/sync_manager.go

```diff
@@ -233,7 +234,8 @@ func (s *SyncManager) CheckPastBeacons(ctx context.Context, upTo uint64, cb func
 		}
 		// verify the signature validity
 		if err = s.scheme.VerifyBeacon(b, s.info.PublicKey); err != nil {
-			logger.Errorw("invalid_beacon", "round", b.Round, "err", err)
+			// this is not to be logged as an error since the goal here is to detect invalid beacons.
```

Comment:
why would error logging prevent this?

Reply:
It doesn't prevent it but it makes e.g. monitoring system fire up on an expected error instead of on an actual error.